tests/conftest.py

```diff
@@ -102,6 +102,14 @@ def pytest_addoption(parser):
              'SSH server on your machine. In certain environments, this '
              'may be insecure! Default: False'
     )
+    test_selection_group.addoption(
```

Comment:
Is this change intentional? This seems unrelated to the rest of the changes here.

Reply:
It was intentional, it fixes pytest for salt, so you can run integration tests.

I can move it to a new PR if you want.

Reply:
Nah, it's good. Just making sure.